app/src/main/java/org/simple/clinic/summary/PatientSummaryUpdate.kt

```diff
@@ -22,7 +22,7 @@ class PatientSummaryUpdate : Update<PatientSummaryModel, PatientSummaryEvent, Pa
       is CurrentFacilityLoaded -> next(model.currentFacilityLoaded(event.facility))
       PatientSummaryEditClicked -> dispatch(HandleEditClick(model.patientSummaryProfile!!))
       is PatientSummaryLinkIdCancelled -> dispatch(HandleLinkIdCancelled)
-      is ScheduledAppointment -> scheduleAppointmentSheetClosed(event, model)
+      is ScheduledAppointment -> dispatch(TriggerSync(event.sheetOpenedFrom))
```

Comment:
This will get triggered even if the user doesn't schedule an appointment but the schedule sheet is closed right?

Reply:
It won't. The `ScheduledAppointment` event is triggered only if an appointment is scheduled.

Reply:
What if a BP or Blood sugar is recorded and the appointment is not scheduled? Will we sync at regular intervals like before?

Reply:
Yes, the regular syncs will continue to happen. This PR only adds additional sync triggers.